site/content/en/docs/Concepts/accounts.md

```diff
@@ -46,23 +46,70 @@ or workflows run.
 
+#### Specifying Subnets
+
+When using an existing VPC you may need to specify which subnets of the VPC can be used for infrastructure. This is useful
+when only some private subnets have internet routing. To do this you can supply a comma separated list of subnet IDs using
+the `--subnets` flag, or repeat the flag multiple times. For example:
+
+`agc account activate --vpc my-existing-vpc-id --subnets subnet-id-1,subnet-id-2 --subnets subnet-id-3`
+
+We recommend a minimum of 3 subnets across availability zones to take advantage of EC2 instance availability and to
+ensure high availability of infrastructure.
+
+#### Using Only Public Subnets
+
+Amazon Genomics CLI can create a new VPC with only public subnets to use for its infrastructure using the `--usePublicSubnets` flag.
+
+`agc account activate --usePublicSubnets`
+
+{{% alert title="Warning" color="warning" %}}
+Currently, use of public subnets is only supported for contexts that use the Cromwell or Nextflow engines.
+{{% /alert %}}
+
+This can reduce costs by removing the need for NAT Gateways and VPC Gateway Endpoints to route internet traffic from private subnets.
+It will also reduce the number of Elastic IP Addresses consumed by your infrastructure. When using a VPC with only public
+subnets you will need to ensure that the contexts defined in `agc-project.yaml` files declare that they
+will use public subnets. For example:
+
+```yaml
+contexts:
+  myContext:
+    usePublicSubnets: true
+    engines:
+      - type: wdl
+        engine: cromwell
+```
+
+##### Security Considerations
+
+Although your infrastructure will be protected by security groups you should be aware that any manual modification of these may result in exposing your
+infrastructure to the internet. For this reason *we do **not** recommend using this configuration in production 
+or with sensitive data*.
+
+#### Updating
+
 Issuing account activate commands more than once effectively updates the core infrastructure with the difference between
 the two commands. For example, if you had previously activated the account using `agc account activate` and later invoked
 `agc account activate --bucket my-existing-bucket --vpc my-existing-vpc-id` then Amazon Genomics CLI will update to use `my-existing-bucket`
 and the identified VPC. The old VPC and S3 buckets will be *retained* according to their retention policy.
 
 If you initially activated the account with `agc account activate --bucket my-existing-bucket --vpc my-existing-vpc-id` 
 and later invoked `agc account activate` then Amazon Genomics CLI will stop using the previous specified bucket, however the VPC will 
-be recalled and re-used. *ALL* of the pre-existing S3 and VPC infrastructure will be retained and a new bucket will be created for use by Amazon Genomics CLI.
+be recalled and re-used. *ALL* the pre-existing S3 and VPC infrastructure will be retained and a new bucket will be created for use by Amazon Genomics CLI.
```

Comment:
Will it also re-use subnets from the previous activation?
What is expected behavior in the opposite scenario:
1) agc account activate
2) agc account activate -vpc <vpc-id> -subnets <subnet-ids>

Another scenario, when a customer activated an account with custom vpc, but didn't specify any subnets, later they realized that subnet specification is required, and re-activated the account, but this time with subnets:
1) aws account activate -vpc ...
2) aws account activate -vpc ... -subnets ...